envoy/go.mod

```diff
@@ -1,9 +1,13 @@
 module github.com/envoyproxy/go-control-plane/envoy
 
-go 1.21
+go 1.22.8
+
+// Used to resolve import issues related to go-control-plane package split (https://github.com/envoyproxy/go-control-plane/issues/1074)
+replace github.com/envoyproxy/go-control-plane@v0.13.2 => ../
```

Comment:
Is there a reason for pinning the version here but not in the go.mod files for `examples/dyplomat` and `ratelimit` packages?